### PR TITLE
Feature/ayush khandelwal 007/metrics overhaul

### PR DESCRIPTION
**Pull request by Ayush-Khandelwal-007**
**Description**

This PR is related to the issue in the extension, that is the sparkling graphs for metrics in visualise drawer
https://github.com/layer5labs/meshery-extensions/issues/29

**Notes for Reviewers**
Changes are not complete yet designs are yet to be made and implemented but the basic skeleton is changed.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/GrafanaCustomChart.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ const grafanaStyles = (theme) => ({
   },
   card: {
     height: '100%',
-    width: "100%"
+    width: "100%",
+    background:'#f3efff',
```
Comment (dhruv0000) [inline ui/components/GrafanaCustomChart.js:48]: image

Are we sure, we want to go with purple?

Light green/blue might be bettor option considering the Brand Guide.

I tried out few colors that might be better aligned with the style.
(#eaf8f5)
image

or (#eaf6fc)
image

What do you think, @Ayush-Khandelwal-007 ?

**Review by dhruv0000 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Pretty please with sugar on top, will you try selections from the Meshery color scheme to see if one of the these colors works? 

https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=65%3A6

Screen Shot 2021-07-07 at 7 44 03 PM

**Comment by Ayush-Khandelwal-007** [comment]
I have made few changes. Ill just rebase my branch and look if everything is fine and push by tonight. @leecalcote 👀 there is something that you might like to see here(something that you were waiting for).

**Comment by Ayush-Khandelwal-007** [comment]
@dhruv0000 What if we let the cards be white itself and give bg colour to the accordion itself? 👀

**Comment by l5io** [comment]
🚀 Preview for commit 59b2e37f4a5484b7ec9ee400c7c6038356a1add3 at: https://60e86983ad421f2df6cb1408--meshery-docs-preview.netlify.app

**Comment by Ayush-Khandelwal-007** [comment]
https://user-images.githubusercontent.com/59086855/125106299-1e062900-e0fd-11eb-8a18-43624a500a6b.mov

@leecalcote Is this something you were waiting for?

**Comment by leecalcote** [comment]
Very nice! Yes, you've nailed it in this screen recording. Those tickers are exactly it.

**Pull request closed without merging**